pudb/debugger.py

```diff
@@ -342,6 +344,24 @@ def set_frame_index(self, index):
 
         self.ui.stack_list._w.set_focus(self.ui.translate_ui_stack_index(index))
 
+    def open_file_to_edit(self, filename, line_number):
```

Comment:
No real reason for this to be a method. It doesn't use `self` AFAICT. Move to a function?

Reply:
DONE